apps/demos/Demos/Chat/Customization/Vue/data.ts

```diff
@@ -1,7 +1,9 @@
+import type { DxChatTypes } from "devextreme-vue/chat";
+
 const date = new Date();
 date.setHours(0, 0, 0, 0);
 
-function getTimestamp(date, offsetMinutes = 0) {
+function getTimestamp(date: Date, offsetMinutes: number = 0): number {
```

Comment:
There is no need to specify the type here, as it will be automatically derived from the default value.
```suggestion
function getTimestamp(date: Date, offsetMinutes = 0): number {
```